modules/oc-compliance-fetching-compliance-remediation-details.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * security/oc_compliance_plug_in/oc-compliance-plug-in-using.adoc
+
+[id="fetching-compliance-remediation-details_{context}"]
+= Fetching compliance remediation details
+
+The Compliance Operator provides remediation objects that are used to automate the changes required to make the cluster compliant. The `fetch-fixes` subcommand  can help you understand exactly which configuration remediations are used. Use the `fetch-fixes` subcommand to extract the remediation objects from a profile, rule, or `ComplianceRemediation` object into a directory to inspect.
+
+.Procedure
+
+. View the remediations for a profile:
++
+[source,terminal]
+----
+$ oc compliance fetch-fixes profile ocp4-cis -o /tmp
+----
++
+.Example output
+[source,terminal]
+----
+No fixes to persist for rule 'ocp4-api-server-api-priority-flowschema-catch-all' <1>
+No fixes to persist for rule 'ocp4-api-server-api-priority-gate-enabled'
+No fixes to persist for rule 'ocp4-api-server-audit-log-maxbackup'
+Persisted rule fix to /tmp/ocp4-api-server-audit-log-maxsize.yaml
+No fixes to persist for rule 'ocp4-api-server-audit-log-path'
+No fixes to persist for rule 'ocp4-api-server-auth-mode-no-aa'
+No fixes to persist for rule 'ocp4-api-server-auth-mode-node'
+No fixes to persist for rule 'ocp4-api-server-auth-mode-rbac'
+No fixes to persist for rule 'ocp4-api-server-basic-auth'
+No fixes to persist for rule 'ocp4-api-server-bind-address'
+No fixes to persist for rule 'ocp4-api-server-client-ca'
+Persisted rule fix to /tmp/ocp4-api-server-encryption-provider-cipher.yaml
+Persisted rule fix to /tmp/ocp4-api-server-encryption-provider-config.yaml
+----
+<1> The `No fixes to persist` warning is expected whenever there are rules in a profile that do not have a corresponding remediation, because either the rule cannot be remediated automatically or a remediation was not provided.
+
+. You can view a sample of the YAML file. The `head` command will show you the first 10 lines:
++
+[source,terminal]
+----
+$ head /tmp/ocp4-api-server-audit-log-maxsize.yaml
+----
++
+.Example output
+[source,terminal]
```

Comment:
Are the YAML outputs for these examples listed as terminal because of the display?

Reply:
Yes, terminal is correct